toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -101,7 +101,11 @@ done
 [[ -z "${repo_path}" ]] && repo_path=${script_dir} && repo_path=${repo_path%'/toolkit'*}
 [[ ! -d "${repo_path}" ]] && { print_error " Directory ${repo_path} does not exist"; exit 1; }
 [[ -z "${mode}" ]] && mode="build"
-[[ -z "${version}" ]] && version="2.0"
+[[ -z "${version}" ]] && version="3.0"
+# TODO: Remove when PMC is available for 3.0
+if [[ "${version}" == "3.0" ]]; then
```

Comment:
Also for Mariner 2.0 (and eventually for 3.0) there is a DAILY_BUILD_REPO argument that we can use.  So we now have access to the daily repos for that too.

The difference here is that a repo file must be provided.  That repo file is generated as part of the daily build and is shared in a daily artifact feed.

Reply:
Mostly this is FYI for 2.0 but it will also be in 3.0 soon.